mypy/main.py

```diff
@@ -222,6 +222,11 @@ def process_options(args: List[str],
     # which will make the cache writing process output pretty-printed JSON (which
     # is easier to debug).
     parser.add_argument('--debug-cache', action='store_true', help=argparse.SUPPRESS)
+    # --half-assed mode is a bit like --silent-imports in that it distinguishes
+    # modules listed on the command line; however instead of ignoring such
+    # imports, it follows them but silences errors.  Stubs are unaffected.
```

Comment:
Perhaps mention how blocker errors work in `--half-assed` mode. What if we have both `--half-assed` and `--silent-imports` -- would we then follow imports (but ignore errors) if we can find an implementation of a module not listed on the command line, and ignore the import silently if no implementation can be found?